### PR TITLE
Release 2019-02-12 - Hotfix Release

### PR DESCRIPTION
**Pull request by jnoller**
CVE-2019-5736 mitigation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,48 @@
...
 
+### Release 2019-02-12 - Hotfix Release
+
+**Hotfix releases follow an accelerated rollout schedule - this release should be in all regions by 12am PST**
+
+* Kubernetes 1.12.5, 1.11.7, 1.10.12, 1.9.11 released (1.8 is deprecated, please upgrade to 1.9.x or higher)
+* This release mitigates CVE-2019-5736 for Azure Kubernetes Service (see below).
+    * Please note that GPU-based nodes do not support the new container runtime yet. We will provide another service update once a fix is available for those nodes.
+
+**CVE-2019-5736 notes and mitigation
```
Comment (seanmck) [inline CHANGELOG.md]: Missed closing ** here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,15 +2,48 @@
 
 ## Releases
 
+### Release 2019-02-12 - Hotfix Release
+
+**Hotfix releases follow an accelerated rollout schedule - this release should be in all regions by 12am PST**
```
Comment (seanmck) [inline CHANGELOG.md]: For complete clarity, maybe put a date next to 12am PST? I know it's implied by the date in the release name above, but people might not pick up on that.

File: CHANGELOG.md (hunk 1 above)
Comment (jnoller) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (jnoller) [reply CHANGELOG.md]: fixed

**Pull request merged**